src/app/components/Grid/index.jsx

```diff
@@ -30,7 +30,7 @@ export const GhostGelPageGrid = styled(Grid)`
   background-color: ${C_GHOST};
 `;
 
-export const StyledFrontPageMain = styled.main`
+export const StyledFrontPageSection = styled.section`
```

Comment:
Can this be removed to the `FrontPageSection`? It is not linked with the Grid anymore.

Reply:
This has been moved to `src/app/containers/FrontPageSection/index.js`